### DISCUSSION TITLE
broker load failed ：errors while write data to output stream,cause by : null

### DISCUSSION THREAD

**Discussion by fisher997** [apache/doris#21122] [A - General / Q&A]
版本：1.2.4
生产环境每天几百张表hdfs broker load 到 doris ， 会有几个load 任务报错 failed ：errors while write data to output stream,cause by : null

**Comment by yagagagaga** [comment]
Can you show the complete log of FE and BE?

**Reply by fisher997** [reply]
内网日志拿不出来 ， 这个问题很奇怪 ， 过一会重新执行这个 load 就会成功 ， 几乎每天都会出现

**Reply by fisher997** [reply]
昨天看了下 fe 和 be 的日志 ， 没看到有什么异常 ， broker 的日志里面有报错

**Reply by yagagagaga** [reply]
Can you show the full stack of broker exception in log?
